awswrangler/opensearch/_utils.py

```diff
@@ -0,0 +1,107 @@
+"""Amazon OpenSearch Utils Module (PRIVATE)."""
+
+from typing import Optional
+
+import boto3
+import logging
+
+from awswrangler import _utils, exceptions
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+from requests_aws4auth import AWS4Auth
+
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+def _get_distribution(client: Elasticsearch):
+    return client.info().get('version', {}).get('distribution', 'elasticsearch')
+
+
+def _get_version(client: Elasticsearch):
+    return client.info().get('version', {}).get('number')
+
+
+def _get_version_major(client: Elasticsearch):
+    version = _get_version(client)
+    if version:
+        return int(version.split('.')[0])
+    return None
+
+
+def connect(
+    host: str,
+    port: Optional[int] = 443,
+    boto3_session: Optional[boto3.Session] = boto3.Session(),
+    region: Optional[str] = None,
+    fgac_user: Optional[str] = None,
```

Comment:
Nitpick - it's probably fine to use just `username` & `password` if we clarify in the docs that this will only work with fine-grained access control enabled.

Reply:
I don't want to limit data-wrangler to only FGAC clusters. from my experience the majority of implementations are not FGAC enabled. 
Furthermore, FGAC is only available from ES >= 6.7

Reply:
I only meant the naming of parameters - we definitely should support both

Reply:
got it. you're right. renamed